main.tf

```diff
@@ -31,6 +36,25 @@ data "aws_subnet" "selected" {
   id = var.subnet_ids[0]
 }
 
+################################################################################
+# Global Database
+################################################################################
+
+data "aws_kms_alias" "rds" {
```

Comment:
I think it makes more sense to make `kms_key_id` a mandatory variable and force people to use Customer managed keys (CMK) (or if they really want specify the arn of the aws/rds alias). It's best practise (and for some of our customers mandatory) to use CMK's anyway.

Reply:
Basically tried to "simplofy" the setup. You can still pass your KMS key if you want yours. But this mimics the same behaviour as a normal (or the primary cluster). If you don't specifify a KMS key it uses the AWS alias

Reply:
discussed offline